stream/feed.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, client, feed_slug, user_id, token):
         self.slug = feed_slug
         self.user_id = str(user_id)
         self.id = "%s:%s" % (feed_slug, user_id)
-        self.token = token
+        self.token = token.decode('utf-8')
```

Comment:
This isn't forward compatible. Please do this:

```suggestion
        self.token = token.decode('utf-8') if isinstance(token, bytes) else token
```